src/core/readyException.js

```diff
@@ -0,0 +1,13 @@
+define( [
+	"../core"
+], function( jQuery ) {
+
+"use strict";
+
+jQuery.readyException = function( error ) {
+	window.setTimeout( function() {
+		throw error;
+	} );
```

Comment:
It seems we would need to rename `readyException`? If it would be used in more general sense? Also we need to re-throw error twice?